dojson/cli/__init__.py

```diff
@@ -30,8 +34,17 @@
 
 .. code-block:: console
 
-    $ dojson -l marcxml -d marcxml do marc21 do to_marc21 < in.xml > out.xml
-    $ diff in.xml out.xml
+    $ dojson -l marcxml -d marcxml do marc21 do to_marc21 < tests/data/test_1.xml > output.xml
+    $ xmllint --format output.xml | diff - tests/data/test_1.xml
+
+To get a list of fields that are missing mappings use the ``missing`` command:
```

Comment:
The word "mappings" may be misleading here. People may think of Elasticsearch mappings.  Better say "rules" or "conversion rules".  Let's use the same vocabulary everywhere.